### ISSUE TITLE
Publish will not start if using monorepo + --ignore flag

### ISSUE THREAD

**Issue by novaknole** [changesets/action#429]
I am using `changeset version --ignore` in my monorepo which successfully DOES NOT include the removal of `.changeset` files for those packages that have been ignored. 

This is expected, because PR that changeset/action creates must only include:
* those `readme.md` changes
* Removal of `.changeset` files that were not ignored.

You might think that once this PR is merged, publish would start. The problem is that there're still other `.changesets`  files(those that were ignored).  The action code comes [here](https://github.com/changesets/action/blob/31ff97e8acd3cfd08714eab3721f55da10003255/src/index.ts#L53), and realizes that `changesets.length` is more than 0, so doesn't run the publish task, instead it again tries to create another PR.

I am not sure how you solve it. The only way I can think of is to publish the package on my own(not use the action of changeset for it). I can clearly do that, but then I also have to add another task to create a release and tag. This is also not a problem, but if I have to write all this code on my own, using changeset kind of loses its essence.  

Am I missing something ?

**Comment by Fart10** [comment]
> I am using `changeset version --ignore` in my monorepo which successfully DOES NOT include the removal of `.changeset` files for those packages that have been ignored.
> 
> This is expected, because PR that changeset/action creates must only include:
> 
> * those `readme.md` changes
> * Removal of `.changeset` files that were not ignored.
> 
> You might think that once this PR is merged, publish would start. The problem is that there're still other `.changesets` files(those that were ignored). The action code comes [here](https://github.com/changesets/action/blob/31ff97e8acd3cfd08714eab3721f55da10003255/src/index.ts#L53), and realizes that `changesets.length` is more than 0, so doesn't run the publish task, instead it again tries to create another PR.
> 
> I am not sure how you solve it. The only way I can think of is to publish the package on my own(not use the action of changeset for it). I can clearly do that, but then I also have to add another task to create a release and tag. This is also not a problem, but if I have to write all this code on my own, using changeset kind of loses its essence.
> 
> Am I missing something ?

**Comment by Fart10** [comment]
Hello world
